chargeOclock/devTemp/devClientsViewController.swift

```diff
@@ -47,4 +49,18 @@ class devClientsViewController: UITableViewController {
 		}
 	}
 
+	func deleteTapped(indexPath: IndexPath) {
+		dataSource.clients[indexPath.row].delete { error in
+
+			if let error = error {
+				print("💥 \(error)")
```

Comment:
print out the `localizedDescription` for the error...

also, showing something to the user might be a nice idea 😉